Source/WebCore/platform/mediastream/gstreamer/GStreamerMediaStreamSource.cpp

```diff
@@ -635,25 +635,47 @@ static GstStateChangeReturn webkitMediaStreamSrcChangeState(GstElement* element,
 {
     GST_DEBUG_OBJECT(element, "%s", gst_state_change_get_name(transition));
     WebKitMediaStreamSrc* self = WEBKIT_MEDIA_STREAM_SRC_CAST(element);
-    GstStateChangeReturn result = GST_ELEMENT_CLASS(webkit_media_stream_src_parent_class)->change_state(element, transition);
+    GstStateChangeReturn result;
+    bool noPreroll = false;
```

Comment:
Nit: this variable name could be a better one. Which one? Your call :relaxed:

Reply:
I chose this name because it matches `GST_STATE_CHANGE_NO_PREROLL`. Dunno what name I should use.